Traceback (most recent call last):
  File "F:\mygit\python\exchange_dj\index\api\api.py", line 48, in inn
    data = fun(*args,**kw)
  File "F:\mygit\python\exchange_dj\index\api\ok\ok_api.py", line 95, in depth
    return [data['asks'][::-1],data['bids']]
KeyError: 'asks'
